vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/SparqlQueryResultDataSource.java

```diff
@@ -164,4 +163,9 @@ public String toString() {
 				+ ", queryBody=" + queryBody + "]";
 	}
 
+	@Override
```

Comment:
add javadoc

Reply:
we don't add javadoc for overriden methods right ? just once in the interface

Reply:
It depends, if you want to be specific make explicit what you change when you override, you should add a javadoc. I guess in this case it is not necessary, since we don't even have a specific wiki page for the syntax of each DataSource.